src/staff-administrator/components/staff-add-edit-user/staff-user-check-answers/staff-user-check-answers.component.html

```diff
@@ -1,15 +1,140 @@
-<exui-page-wrapper [title]="'Check your answers'" [backLink]="'/staff/add-user'">
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-full">
-      <button class="govuk-button govuk-!-margin-right-1 govuk-!-margin-bottom-0"
-        [routerLink]="'/staff'"
-      >
-        Continue
-      </button>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <exui-page-wrapper [title]="'Check your answers before adding a new user'" [backLink]="'/staff/add-user'">
```

Comment:
You have removed `<exui-page-wrapper>` closing tag. It will throw an error and the page will load.